backend/migrations/versions/d65d4263d860_add_date_assigned_to_usergoal_primary_.py

```
@@ -0,0 +1,67 @@
+        sa.Column("user_id", sa.Integer(), nullable=False),
+        sa.Column("goal_id", sa.Integer(), nullable=False),
+        sa.Column("date_assigned", sa.DateTime(), nullable=False),
+        sa.Column("date_completed", sa.DateTime(), nullable=True),
+        sa.ForeignKeyConstraint(["goal_id"], ["goals.id"]),
+        sa.ForeignKeyConstraint(["user_id"], ["users.id"]),
+        sa.PrimaryKeyConstraint("user_id", "goal_id", "date_assigned"),
+    )
+
+    # Copy data from the old table to the new table
+    op.execute(
+        "INSERT INTO user_goals_new (user_id, goal_id, date_assigned, date_completed) SELECT user_id, goal_id, date_assigned, date_completed FROM user_goals"
```

Ruff (0.8.0)

Trailing comma missing

Add trailing comma

(COM812)